src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -196,6 +196,26 @@
 					$( '.give-fee-recovery-donors-choice' ).toggleClass( 'active' );
 				} );
 
+				// If mailchimp is opted in by default, add border on load
+				if ( $( 'input[name="give_mailchimp_signup"]' ).prop( 'checked' ) === true ) {
```

Comment:
@henryholtgeerts can this be written dynamically so there's no repeated code?

Reply:
Added a setupCheckbox function to handle listeners/setup of these kind of prominent checkboxes, ref: https://github.com/impress-org/givewp/commit/7b33419eb8aeea604596678716660c0d6b0a1ce5